controller/controller.go

```diff
@@ -580,7 +582,7 @@ waitForLock:
 				select {
 				case <-timeout.C:
 					for _, podToDelete := range newPods {
-						err := deletePod(logger, mr, clientset, redisClient.Client, schedulerName, podToDelete.GetName())
+						err := deletePod(logger, mr, clientset, redisClient.Client, schedulerName, gameName, podToDelete.GetName(), "update")
```

Comment:
It is better if the reasons are at models/contants.go and you call the variables

Reply:
This reason should not be update. At this point, update timed out and is deleting the newly created pods. Probably an error of ErrImagePull occurred or the pods are pending. A better reason would be "update error"